block/pkg/datastore/init.go

```diff
@@ -0,0 +1,5 @@
+package datastore
+
+import (
+	_ "github.com/opensds/multi-cloud/block/pkg/datastore/aws/"
```

Comment:
Why do we use Non-Canonical imports here?
What we are importing using -> github.com/opensds/multi-cloud/block/pkg/datastore/aws/  ?
It should be github.com/opensds/multi-cloud/block/pkg/datastore/aws

Reply:
This import is for executing the init() function of that import. This will basically call aws->init() to register driver